src/shared-components/immersiveModal/style.ts

```diff
@@ -219,7 +218,7 @@ export const MainModalContentContainer = styled.div<HasHeaderImageProps>`
   position: relative;
   border-top-left-radius: 32px;
   border-top-right-radius: 32px;
-  box-shadow: 0px -8px 24px rgba(51, 46, 84, 0.05);
+  box-shadow: ${({ theme }) => theme.BOX_SHADOWS.modalMobile};
```

Comment:
This looks fine, but L227 below (`box-shadow: 0px 8px 24px rgba(51, 46, 84, 0.05);`) looks like we should pull in `theme.BOX_SHADOWS.clickable`, perhaps?

Reply:
no that is modalDesktop we are expecting an answer from Ben. I think it willl be removed

Reply:
Removed modal desktop boxShadow we only have 1 shadow for modal now and is theme aware